.github/workflows/CI.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+# Run on master, any tag or any pull request
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+  pull_request:
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.3'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+        include:
+          # Add a 1.0 job just to make sure we still support it
+          - os: ubuntu-latest
+            version: 1.0.5
```

Comment:
Could using `1` or `1.0` allow us to use the latest 1.0.x version instead of doing it manually on any new versions?

Reply:
That doesn't work. The trailing `.0` seems to get stripped and run `1` which would be `1.5` according the `setup-julia` action.